src/pages/Host/LiveSession/HostLiveSession.css

```
@@ -1,41 +1,655 @@
+  font-weight: 700;
+  color: #fff;
+  flex-shrink: 0;
+  overflow: hidden;
+}
+.host-live-chat-avatar img { width: 100%; height: 100%; object-fit: cover; }
+.host-live-chat-body { flex: 1; min-width: 0; }
+.host-live-chat-meta { display: flex; align-items: center; gap: 5px; margin-bottom: 1px; }
+.host-live-chat-name { font-size: 12px; font-weight: 700; color: var(--neutral-900); }
+[data-theme="dark"] .host-live-chat-name { color: #fff; }
+.host-live-chat-time { font-size: 10px; color: var(--neutral-400); }
+.host-live-chat-text { font-size: 13px; color: var(--neutral-700); word-break: break-word; line-height: 1.45; }
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)